sdk/lib/isolate/isolate.dart

```diff
@@ -207,7 +207,9 @@ final class Isolate {
   /// (see http://dartbug.com/36983) or, if the state includes objects that
   /// can't be spent between isolates, a runtime failure.
   ///
-  /// ```dart import:convert import:io
```

Comment:
This style is deliberate. 

The import declarations are not intended to be part of the example code that is visible in the generated DartDoc.

The 
````
```dart import:convert import:io
````
are not intended as imports, but as instructions to our own "code sample validator" tool, which analyzes the sample in a context where those two imports are assumed to have been performed.